tools/harness-automation/doc/quickstart.rst

```diff
@@ -0,0 +1,87 @@
+Thread Harness Automation Quick Start
+=====================================
+
+Setup
+-----
+
+#. Install Latest Thread Harness
```

Comment:
Indicate the exact version of the Thread harness that this tool was tested against.